infra/modules/gcp-host/main.tf

```diff
@@ -160,7 +143,7 @@ module "api_connector" {
 
   secret_bindings = merge(
     # bc some of these are later filled directly, bind to 'latest'
-    { for k, v in module.secrets[each.key].secret_bindings : k => merge(v, { version_number : "latest" }) },
```

Comment:
Here all secrets created were added as part of security bindings, being available as it were env vars